util/cron/test-perf.cray-cs-hdr.champs-c.bash

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+#
+# Run CHAMPS testing on a cray-cs with HDR IB and C backend
+
+CWD=$(cd $(dirname $0) ; pwd)
+
+export CHPL_TEST_PERF_CONFIG_NAME='16-node-cs-hdr'
+export CHPL_NIGHTLY_TEST_CONFIG_NAME="perf.cray-cs-hdr.champs-c"
```

Comment:
For other jobs we used `c-backend` since a lone `c` might not be obvious. No strong preference, but might be nice to unify here